update-merge-rules/index.js

```diff
@@ -0,0 +1,57 @@
+/* 
+This script is meant for a one time use to update the state of repos to disallow squash and merge.
+To use, add .env with GITHUB_TOKEN
+*/
+
+const ISOMER_ADMIN_REPOS = [
+  "isomercms-backend",
+  "isomercms-frontend",
+  "isomer-redirection",
+  "isomerpages-template",
+  "isomer-conversion-scripts",
+  "isomer-wysiwyg",
+  "isomer-slackbot",
+  "isomer-tooling",
+  "generate-site",
+  "travisci-scripts",
+  "recommender-train",
+  "editor",
+  "ci-test",
+  "infra",
```

Comment:
missing isomercms-infra too

Reply:
Thanks for catching, https://github.com/isomerpages/isomer-tooling/pull/47/commits/77e3b2719af3e9eab5ae1ddc017c0f010cc387b0!